esmvalcore/config/_config.py

```diff
@@ -0,0 +1,130 @@
+"""Functions dealing with config-user.yml / config-developer.yml."""
```

Comment:
Note to reviewer: this file was moved here from `esmvalcore/_config/_config.py`, but because all the old code for reading config-user.yml was removed from it, git doesn't recognize it.